src/styles/annotator/bucket-bar.scss

```diff
@@ -6,14 +6,19 @@
 // precedence over the CSS reset applied to `.annotator-frame`.
 .annotator-frame {
   .annotator-bucket-bar {
-    background: rgba(0, 0, 0, 0.08);
+    background: var.$grey-2; // When sidebar is unfolded, remove the background transparency
     pointer-events: none;
     position: absolute;
     height: 100%;
     width: var.$bucket-bar-width;
     left: -(var.$bucket-bar-width);
   }
 
+  // When the sidebar is collapsed, make the background semi-transparent so the text is visible throughout (useful for pages with tight margins)
```

Comment:
Tiny nit: can you break this comment to two lines? We aren't rigorous about exactly 80-character lines, but this one runs fairly long.